documentation/using_plugins.md

```diff
@@ -265,78 +265,7 @@ google_compute_instance.app.1:
 ```
 
 ## AWS Inventory plugin
```

Comment:
Looks like we have not capped "inventory" in the past, but feel free to override the suggestion if that's not the case.